docs/_files/input_parameters_list.csv

```diff
@@ -1,32 +1,32 @@
-Security of supply, e.g. existence of back-up system,,,,Technical,,hidden,,,there are many components and constraints for a flexible backup system,,
-Balancing ratio for anciliary services,Balancing ratio for ancilliary services (need a more precise definition as PF could not re-figure out what this means),,,array,Technical,,hidden,,,?,,
-Number of future extra users,Number of future extra user in energy cell,,,int,Technical,,hidden,,,?,,
-Load types,Designation of flexible and non-flexible loads in the load profile,,,bool,Technical,,data,,OP_IN_21,dsm functionalities exist,,
-Costs of asset use,The price of a liter of diesel or a kWh from the grid,EUR,Value provided,float,Financial,,component,,OP_IN_07,,,To be able to compute the cost associated with the use of an asset
-Project lifetime,The lifetime of the project (simulation time),Time,,float,Simulation,,project,,,,,
+label,:label_DE:,:Definition_DE:,:Definition:,:Unit:,:Default:,:Type:,:Category:,:Restrictions:,internal_categorization,tooltip,Source/related requirements,Ease of oemof implementation,ref,Rational
+Demand profile,Lastzeitreihe,Die Bedarfszeitreihe des Energieverbrauchs mit Zeitstempel,The energy demand time serie with timestep (if different energy types are considered different profiles must be given),,,timeseries,Technical,,data,,,component/constraint exists,,
+Energy timeseries,Feed-in Zeitreihen,"Spezifische Erzeuger/Feed-in-Zeitreihen z.B. von Wind, PV, Hydro","Specific timeseries of energy sources from a primary-secondary transformation (e.g. PV, Wind)",kWh,,timeseries,Technical,between 0 and 1 (normalized with maximum),data,,OP_IN_07,component/constraint exists,,
+Source power mix,Anteil Energieerzeuger am Energiemix,Anteil jedes Energieerzeugers am Energiemix (inkl. % Netzbezug),Percentage of each source of power (inclusive % from Grid),Percentage,,array,Technical,,constraint,,OP_IN_05,component/constraint exists,,
+Lower bound renewable energy share ,min. EE-Anteil,"Eine Randbedingung, welche den minimalen Anteil EE-Energieerzeuger im Bezug auf den optimierten Energiemix festlegt",A constraint that specifies the lower bound on the share of renewable energies in the optimized output energy mix,Percentage,None,float,Technical,,constraint,,,component/constraint exists,,
+CO2 upper limit,max. CO2-Emissionen,"Eine Randbedingung, welche die maximale Masse an CO2-Emissionn festlegt",A constraint that specifies an upper bound on the CO2 emissions,T of CO2,None,float,Environmental,,constraint,,,component/constraint exists,,
+Energy sources capacities,Erzeugerkapazit�ten,"Erzeugertechnologien, die Entscheidungsvariablen der Optimierung sind mit bereits installierte Kapazit�ten",Energy sources with their capacity open for optimization ,kW,None,float,Technical,,component,,,?,,A user can select 
```

Comment:
Looks like an Encoding problem. I edited the csv file using Excel. Shall I change the encoding before saving the file?

Reply:
Excel is bad at editing csv which should be reusable/reopenable by other softwares than Excel ... (probably a strategy?) It is better to use LibreOffice to edit csv files with a tabular view ;)